### ISSUE TITLE
The `--check-only` option enforces wrapping, which is not mentioned by the help text

### ISSUE THREAD

**Issue by edmorley** [timothycrosley/isort#298]
1) Clone https://github.com/mozilla/treeherder (revision 1c00ccf for reproducibility)
2) `pip install -r requirements/common.txt -r requirements/dev.txt`
3) `isort --check-only treeherder/webapp/api/urls.py`

Expected:
No errors shown - since all imports are correctly sorted, and `isort --help` says:

```bash
  -c, --check-only      Checks the file for unsorted imports and prints them
                        to the command line without modifying the file.
```

Actual:
```bash
~/treeherder $ isort --check-only treeherder/webapp/api/urls.py
ERROR: /home/vagrant/treeherder/treeherder/webapp/api/urls.py Imports are incorrectly sorted.
```

Presumably since the wrapping is being changed:
```bash
~/treeherder $ isort --diff treeherder/webapp/api/urls.py
--- /home/vagrant/treeherder/treeherder/webapp/api/urls.py:before       2015-08-10 18:12:02.451898
+++ /home/vagrant/treeherder/treeherder/webapp/api/urls.py:after        2015-08-10 18:31:24.116318
@@ -6,8 +6,9 @@
 from rest_framework import routers

 from treeherder.webapp.api import (artifact, bug, job_log_url, jobs, logslice,
-                                   note, performance_artifact, performance_data,
-                                   projects, refdata, resultset)
+                                   note, performance_artifact,
+                                   performance_data, projects, refdata,
+                                   resultset)

 # router for views that are bound to a project
 # i.e. all those views that don't involve reference data
```

The help text should either be adjusted to say something like:
`Checks the file for unsorted imports or mis-wrapped imports`
...or else `--check-only` should only show errors if the actual order was wrong. (The latter would probably be more useful for me personally, but I understand it may be more surprising for other people. Perhaps a separate `--dont-change-wrapping` option might help, which could be used for --check-only plus all of the other modes? My use-case is running check-only mode on Travis to ensure people don't regress the import order, but I'm not too fussed about the wrapping style).

Using isort 4.0.0 with Python 2.7.10 on Ubuntu 14.04.

Many thanks :-)

**Comment by timothycrosley** [comment]
Hi @edmorley,

Thanks for the feedback! The truth is, the check option is very simple: it essentially runs isort internally, and if there is *any* change (wrapping, order, space before comma, double entering import) between running isort on the file with your given settings and if there is no change it succeeds otherwise, it fails. I would be open too a more robust implementation for check, but do not have the time currently to do it myself, so in the meantime I will simply update the docs.

Thanks!

~Timothy

**Comment by timothycrosley** [comment]
Fixed in version 4.1.0

Thanks!

~Timothy

**Comment by edmorley** [comment]
Thank you :-)
